src/modules/fancyzones/lib/util.h

```diff
@@ -96,6 +96,32 @@ namespace FancyZonesUtils
             SRCCOPY);
     }
 
+    inline COLORREF HexToRGB(const std::wstring& hex)
+    {
+        std::wstring color = hex;
+        if (hex.size() == 7)
+        {
+            color = hex.substr(1);
+        }
+        else if (hex.size() == 9)
+        {
+            color = hex.substr(3);
+        }
+
+        try
+        {
+            const auto tmp = std::stol(color, nullptr, 16);
+            const auto nR = (tmp & 0xFF0000) >> 16;
```

Comment:
let's not use `auto` for the color channels and explicitly designate the type. also make sure there're no warnings.